rtos/Mutex.h

```diff
@@ -82,7 +82,7 @@ class Mutex : private mbed::NonCopyable<Mutex> {
       Wait until a Mutex becomes available.
 
       @return  status code that indicates the execution status of the function:
-               @a osOK the mutex has been obtained.
+               @a Status code such as osOK indicates that the mutex has been obtained.
```

Comment:
No, the only thing this can return is osOK. See the note below. Same for the other function

Reply:
Fyi @aashishc1988

Reply:
addressed